src/run_ci.py

```diff
@@ -24,9 +23,7 @@ def main():
     args = CiArgs()
     console.configure(level=args.logging_level)
 
-    manifest_type = __get_manifest_type(args.manifest)
-
-    if manifest_type == "Test":
+    if __is_test_manifest(args.manifest.name):
```

Comment:
Can we please do a little bit better than an early return? I'll settle for an `if/else`, but ideally we make two classes `CiTestManifest` and `CiInputManifest` that inherit `CiManifest` and have `CiManifest.from_file(path)` that creates one or the other, then call a base `.check()` method.

Reply:
Added the new classes